source/core/slang-uint-set.h

```diff
@@ -142,6 +304,38 @@ inline void UIntSet::add(UInt val)
     m_buffer[idx] |= Element(1) << (val & kElementMask);
 }
 
+inline void UIntSet::add(const UIntSet& other)
+{
+    auto otherCount = other.m_buffer.getCount();
+    if (this->m_buffer.getCount() < otherCount)
+        resizeBackingBufferDirectly(otherCount);
+
+    for (auto i = 0; i < otherCount; i++)
+        m_buffer[i] |= other.m_buffer[i];
 }
 
+template<typename T>
+List<T> UIntSet::getElements() const
```

Comment:
There are still uses of getElements that I think can be replaced with just iterating the set directly.